hack/build-antrea-windows-all.sh

```diff
@@ -69,7 +69,10 @@ pushd "$THIS_DIR/.." > /dev/null
 NANOSERVER_VERSION=$(head -n 1 build/images/deps/nanoserver-version)
 CNI_BINARIES_VERSION=$(head -n 1 build/images/deps/cni-binaries-version)
 GO_VERSION=$(head -n 1 build/images/deps/go-version)
-WIN_BUILD_TAG=$(echo $GO_VERSION $CNI_BINARIES_VERSION $NANOSERVER_VERSION| md5sum| head -c 10)
+WIN_OVS_VERSION=$(head -n 1 build/images/deps/ovs-version-windows)
+WIN_BUILD_OVS_TAG=$(echo $NANOSERVER_VERSION $WIN_OVS_VERSION | md5sum| head -c 10)
```

Comment:
Will `windows-ovs` be used by antrea-agent Pod directly? If yes, I think a md5sum tag is hard to get which OVS is running. We can use `$NANOSERVER_VERSION-$WIN_OVS_VERSION` as its tag, for example `antrea/windows-ovs:1809-2.17.6`

Reply:
windows-ovs image will be used by antrea-agent pod's container antrea-ovs as windows-ovs:193be75ab2, we can retag it as windows-ovs:latest or tag as suggested with antrea/windows-ovs:1809-2.17.6 , 
but may be we can prefer latest tag

Reply:
"latest" shouldn't be used. Such tag means the image could be updated. It would cause uncertain behavior for a cluster as the antrea-ovs container may use different OVS version whenever it's restarted, and no way to pin to an OVS version, which could incur compatibility issue.